packages/bot/src/transformers/reverse/member.ts

```diff
@@ -31,8 +31,8 @@ export function transformMemberToDiscordMember(bot: InternalBot, payload: Member
     avatar: payload.avatar ? iconBigintToHash(payload.avatar) : undefined,
     permissions: payload.permissions?.toString(),
     communication_disabled_until: payload.communicationDisabledUntil ? new Date(payload.communicationDisabledUntil).toISOString() : undefined,
-    deaf: payload.toggles?.deaf,
-    mute: payload.toggles?.mute,
+    deaf: payload.toggles?.deaf ?? false,
+    mute: payload.toggles?.mute ?? false,
```

Comment:
Why this change? `DiscordMember.deaf` and `DiscordMember.mute` is typed optional, so I don't think this change is necessary?

```suggestion
    deaf: payload.toggles?.deaf,
    mute: payload.toggles?.mute,
```

Reply:
https://discord.com/developers/docs/resources/guild#guild-member-object-guild-member-structure

deaf and mute are not optional, this is one of the (breaking) changes in the pr, and since we have an undefined from the `?.` on toggles we need to default the values